src/llmcompressor/utils/helpers.py

```diff
@@ -1062,7 +1064,7 @@ class DisableKVCache:
     ...     output = model(input)
     """
 
-    def __init__(self, model: torch.nn.Module):
+    def __init__(self, model: PreTrainedModel):
```

Comment:
why this change here? Can `torch.nn.Module`s and `transformers.PreTrainedModel`s be used interchangeably?

Reply:
`PreTrainedModel` inherits from `torch.nn.Module`. This type narrowing relates to the fact that this context is only meant to be used with HF Transformers `PreTrainedModel`s, as it references capabilities specific to models such as having a config and disabling the HF cache

Reply:
oh i didn't know that! ok cool. yeah i like what you've done for var names, `model: PreTrainedModel` and `module: torch.nn.Module`